denguefever_tw/dengue_linebot/views.py

```diff
@@ -316,3 +316,53 @@ def suggestion_list(request):
 def gov_report_list(request):
     context = {'gov_reports': GovReport.objects.all().order_by('-report_time')}
     return render(request, 'dengue_linebot/gov_report_list.html', context)
+
+
+@login_required
+def push_msg_form(request):
```

Comment:
hmm... The django form is hard to customize what i want.
e.g.
The implementation of `CheckboxSelectMultiple` is hard-coded to use a `<ul><li></li></ul>` form.
But i just want multiple `<input type="checkbox">` without list.
I need some time to deal with it. :cry: